homeassistant/components/mobile_app/helpers.py

```diff
@@ -86,6 +94,27 @@ def _decrypt_payload(key: str | None, ciphertext: str) -> dict[str, str] | None:
         return None
```

Comment:
Can this case still happen? Or is that replaced with the raising of `CryptoHelper` ?

Reply:
And if this case can still happen, we shouldn't log a warning unless we tried both decryption approaches.